tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_datatype_test.dart

```diff
@@ -119,6 +119,69 @@ fields:
       });
     });
 
+    group(
```

Comment:
nit: This test group is missing a `when` action declaration

Reply:
You are right, but looking at the file it doesn't seem like this is used for any of the tests, so I will opt to keep the pattern here.